opentech/static_src/src/app/src/components/ListingDropdown/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ListingDropdown extends React.Component {
+    static propTypes = {
+        listRef: PropTypes.object,
+        groups: PropTypes.array,
+    }
+
+    handleChange(e) {
+        const groupHeaderPosition = document.getElementById(e.target.value).offsetTop - 75;
```

Comment:
this 75 is a bit magic...  i'm not sure i'm a fan. There must be a better way?

Reply:
yup, agreed. I'll grab the height of the dropdown container then pass as a prop to `<ListingDropdown>` to use as an offset